metadata.json

```diff
@@ -11,6 +11,10 @@
     {
       "name": "puppetlabs/stdlib",
       "version_requirement": ">= 4.13.1 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs/powershell",
+      "version_requirement": ">= 1.0.0"

```

Comment:
Worth noting that I did in fact test this on many versions of the powershell module starting with 1.0.1 which is the earliest version available on the forge.

Reply:
Just changed the version to `>= 1.0.1` since that's the earliest on the forge.